app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form">
+
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field custom-login">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field custom-login">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio %>
+  </div>
+
+  <div class="field custom-login">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field custom-login">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field custom-login">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <button class="mybtn" type="button"><%= f.submit "Sign up" %><button>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+
+</div>
```

Comment:
- [ ] One of the requirements for this milestone is to ensure that the **_Log In and Sign Up_** pages are identical to the wireframe provided. After the setup, kindly ensure that your **_Log In and Sign Up_** pages are identical to the wireframe provided. 💻 😃 

#### Hint: Pay attention to the width of the input boxes and the buttons. 😎 

### Wireframe Provided

![image](https://github.com/Megagig/Blog-App/assets/77106962/36330738-3649-4d0a-9d53-097d6ff14d28)